compiler/rustc_parse/src/parser/mod.rs

```diff
@@ -1612,7 +1612,7 @@ enum FlatToken {
     /// `AttributesData` is inserted directly into the
     /// constructed `AttrTokenStream` as
     /// an `AttrTokenTree::Attributes`.
-    AttrTarget(AttributesData),
+    Attributes(AttributesData),
```

Comment:
I'd rather rename `AttributesData` to `AttrTargetData`, because it doesn't contain just attributes, it contains the "attribute target" as well (the node to which the attributes are applied).

Reply:
I've gone with `AttrsTarget`.